### DISCUSSION TITLE
how to set custom domain for a specific route

### DISCUSSION THREAD

**Discussion by wassimbj** [vercel/vercel#4207] [General] [answered]
what i want to do here, is let say i have route like this my-website.com/community/2 what i want to do is set a custom domain that points to that route so whenever i visit that custom domain i get that route, also that route can have another routes for e.g: my-website.com/community/2/user/20 (or whatever..), can i do achieve this ?

**edit**: by the way this is not just one domain i can have many users that can set his own custom domain maybe 300 or more, how can i achieve this ?

Thanks guys !

**Comment by coetry** [comment, marked as answer]
Hi @wassimbj you can set redirect rules on custom domains via a `now.json` config file, which you can find a configuration reference for [here](https://vercel.com/docs/configuration#introduction/configuration-reference)

You can use our [API](https://vercel.com/docs/api#endpoints/projects/add-a-domain-to-a-project) to assign up to 50 custom domains for a project. If you need any more than that, you'll need to reach out to sales@vercel.com to discuss your needs.
